pkg/utils/loadtests/controller.go

```diff
@@ -54,16 +53,16 @@ func SelectUsers(userList []User, numberOfUsers, threadCount, maxUsers int) ([]U
 	return selectedUsers, nil
 }
 
-//Indentify CI and get unique Job Name
-func GetJobName()(string){
+// Indentify CI and get unique Job Name
+func GetJobName() string {
```

Comment:
The changes in this file look unrelated to the ticket - I'm guessing they got automatically formatted by the IDE.
Probably best to not modify this file in order to keep the git history clean.

Reply:
done

Reply:
auto formatting again, apologies. Looks ok now.

Reply:
branch updated and formatting changes addressed.